jax_md/test_util.py

```diff
@@ -53,6 +55,42 @@ def load_silica_data() -> jnp.ndarray:
     filename = '/google3/third_party/py/jax_md/tests/data/silica_positions.npy'
     return _load_silica_data(filename)
 
+def _load_nc_data(fn, spatial_dimension, dtype, index):
```

Comment:
Can we name these helper functions something a bit more descriptive?  Maybe something like `_load_elasticity_test_data`?

Reply:
done